policyengine_us/reforms/ctc/eppc/expanded_ctc.py

```diff
@@ -0,0 +1,423 @@
+from policyengine_us.model_api import *
+
+
+def create_expanded_ctc() -> Reform:
+    class ctc_phase_in(Variable):
+        value_type = float
+        entity = TaxUnit
+        label = "CTC phase-in"
+        unit = USD
+        definition_period = YEAR
+        reference = "https://eppc.org/wp-content/uploads/2024/10/Tax-Teams-Comment-on-Working-Families.pdf"
+
+        def formula(tax_unit, period, parameters):
+            tax = tax_unit("income_tax_pre_ctc", period)
+            total_benefits = tax_unit.household("household_benefits", period)
```

Comment:
Can we enumerate these instead? I know the document doesn't list them, but do they really want to subtract e.g. the actuarial value of Medicaid and reimbursement value of NSLP? i'd suggest asking jamie and just listing e.g. snap and tanf (which is what the letter sounds like)

Reply:
Currently we just do SNAP, TANF and Free school meals but we will want to add more in the future